content/kubermatic/main/architecture/concept/kkp-concepts/applications/default-applications-catalog/argocd/_index.en.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "ArgoCD Application"
+date = 2024-01-16T12:57:00+02:00
+weight = 1
+
++++
+
+# What is ArgoCD?
+Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
+
+Argo CD follows the GitOps pattern of using Git repositories as the source of truth for defining the desired application state. Kubernetes manifests can be specified in several ways:
+
+- kustomize applications
+- helm charts
+- jsonnet files
+- Plain directory of YAML/json manifests
+- Any custom config management tool configured as a config management plugin
+
+
+For more information on the ArgoCD, please refer to the [official documentation](https://argoproj.github.io/cd/)
+
+# How to deploy?
+
+Argo CD is available as part of the KKP's default application catalog. 
+It can be deployed to the user cluster either during the cluster creation or after the cluster is ready(existing cluster) from the Applications tab via UI.
+
+* Select the ArgoCD application from the Application Catalog.
+
+![Select ArgoCD Application](/img/kubermatic/common/applications/default-app-catalog/01-select-application-argocd-app.png)
```

Comment:
Check the preview, these images don't seem to work.

Reply:
I had similar experience checking the preview.. But verified the path for other existing images too.. It is same as for newly added. Should work! :)

Reply:
If it's not working on https://deploy-preview-1610--cranky-newton-4e6ed2.netlify.app/, I don't think it will work when deployed :-/ At least I don't see any reason for that.

Reply:
Thanks for the pointer.. Fixed the typo error in the link address. Working fine. Images are getting rendered.